components/post_view/reaction/reaction.jsx

```diff
@@ -247,14 +247,19 @@ export default class Reaction extends React.PureComponent {
                 onEnter={this.loadMissingProfiles}
             >
                 <div
+                    id={`post-reaction-${this.props.post.id}-${this.props.emojiName}`}
```

Comment:
I suggest to change this to:
```suggestion
                    id={`postReaction-${this.props.post.id}-${this.props.emojiName}`}
```